CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Changed
 - Readme - Add a community section.
 - Readme - Remove the Licence section as it is already accessible in the Github page header.
+- Configuration - Add initialisation of configuration directories
```

Comment:
I think we can consider this as a feature, so it should be in the "Added" section:
```markdown
### Added
- Configurations - Users can specify the directory for Forest Smart Implementation.

### Changed
...
```

Reply:
Resolved in 5005b94